pyuvdata/uvdata/ms.py

```diff
@@ -382,7 +394,7 @@ def _write_ms_field(self, filepath):
 
         field_table = tables.table(filepath + "::FIELD", ack=False, readonly=False)
         time_val = (
-            Time(np.median(self.time_array), format="jd", scale="utc").tai.mjd * 86400.0
+            Time(np.median(self.time_array), format="jd", scale="utc").mjd * 86400.0
```

Comment:
Very minor, but I think this could be tied to an astropy unit conversion: `Time().mjd.to(u.s).value`?

Reply:
So as luck has it, the call to `mjd` spits out value rather than a `Quantity`, which means that I can't call the `to` method on it. As such, I've kept this as is for now.